server/src/controllers/Events/resolver.ts

```diff
@@ -323,7 +323,15 @@ export class EventResolver {
     @Arg('limit', () => Int, { nullable: true }) limit?: number,
     @Arg('offset', () => Int, { nullable: true }) offset?: number,
   ): Promise<EventWithChapter[]> {
-    return await prisma.events.findMany({
+    const events = await prisma.events.findMany({
```

Comment:
```suggestion
    return await prisma.events.findMany({
```
Nit: there's no need to create this only to return it, since that's all this function does.